src/api/package.json

```diff
@@ -40,6 +41,7 @@
     "exceljs": "^4.4.0",
     "express-rate-limit": "^7.3.1",
     "jsonwebtoken": "^9.0.2",
+    "mysql2": "^3.10.1",
```

Comment:
will these be removed after launch? ticket?

Reply:
Created a ticket, EMS-3521